specifyweb/frontend/js_src/lib/components/Molecules/ResourceLink.tsx

```diff
@@ -11,6 +11,9 @@ import { getResourceViewUrl } from '../DataModel/resource';
 import { deserializeResource } from '../DataModel/serializers';
 import { ResourceView } from '../Forms/ResourceView';
 
+export const IsNotReadOnly = React.createContext(false);
```

Comment:
(optional) Add a one sentance description of what this context stores/what it should be used for.

Preferably use the `/**  Something */` syntax rather than `/* something */` or `// something`

That special syntax should be used when describing methods, classes, functions and global variables. [See more](https://tsdoc.org/)